test/users.spec.js

```diff
@@ -71,93 +76,174 @@ describe('User', () => {
       email: 'ablaw@test.com',
       password: 'abilaw'
     })
-    .expect(201).end((err, res) => {
+    .expect(201)
+    .end((err, res) => {
       expect(res.body.RoleId).to.equal(2);
       done();
     });
   });
 
   it('should validate that new users create First and Last names.', (done) => {
-    request.get('/api/users/adlaw').set('x-access-token', token).set('Accept', 'application/json')
-    .expect(200).end((err, res) => {
+    request.get('/api/users/adlaw').set('x-access-token', token)
+    .set('Accept', 'application/json').expect(200)
+    .end((err, res) => {
       expect(res.body.lastname).to.equal('Law');
       expect(res.body.firstname).to.equal('Ade');
       done();
     });
   });
 
   it('should ensure new user cannot be created without first and last names.', (done) => {
-    request.post('/api/users').set('x-access-token', token).set('Accept', 'application/json').send({
+    request.post('/api/users').set('x-access-token', token)
+    .set('Accept', 'application/json').send({
       firstname: 'Ade2',
       lastname: '',
       username: '',
       email: 'adlaw2@test.com',
       password: 'adelaw2',
       roleId: 2
-    }).expect(400).end((req, res) => {
+    })
+    .expect(400)
+    .end((req, res) => {
       expect(res.body.message).to.equal('User not created.');
-      expect(((res.body.error).split(','))[0]).to.equal('Validation error: Validation notEmpty failed');
+      expect(((res.body.error).split(','))[0]).to
+      .equal('Validation error: Validation notEmpty failed');
```

Comment:
### Critical

Please make this error message more descriptive of the actual issue